code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -698,23 +701,23 @@
 	ph = 10
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/medicine/oculine/on_mob_life(mob/living/carbon/M)
+/datum/reagent/medicine/oculine/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
 	var/obj/item/organ/eyes/eyes = M.getorganslot(ORGAN_SLOT_EYES)
-	M.adjust_blindness(-2)
-	M.adjust_blurriness(-2)
+	M.adjust_blindness(-2 * REM * delta_time)
+	M.adjust_blurriness(-2 * REM * delta_time)
 	if (!eyes)
 		return
-	eyes.applyOrganDamage(-2)
+	eyes.applyOrganDamage(-2 * REM * delta_time)
 	if(HAS_TRAIT_FROM(M, TRAIT_BLIND, EYE_DAMAGE))
-		if(prob(20))
+		if(DT_PROB(10, delta_time))
 			to_chat(M, "<span class='warning'>Your vision slowly returns...</span>")
 			M.cure_blind(EYE_DAMAGE)
 			M.cure_nearsighted(EYE_DAMAGE)
 			M.blur_eyes(35)
 	else if(HAS_TRAIT_FROM(M, TRAIT_NEARSIGHT, EYE_DAMAGE))
 		to_chat(M, "<span class='warning'>The blackness in your peripheral vision fades.</span>")
 		M.cure_nearsighted(EYE_DAMAGE)
-		M.blur_eyes(10)
+		M.blur_eyes(5)
```

Comment:
It has been two months since I touched this part of the code. Let me check.

Reply:
It looks like I didn't bother. I think the logic here is that, since it's basically a one-time event, it doesn't need to be delta-timed. Like, it's just blurring your eyes when it cures them and it's not exactly doing that over time.